docs/faq.md

```diff
@@ -271,6 +273,55 @@ There are several ways of doing this:
 - [`docker pull`](https://docs.docker.com/engine/reference/commandline/pull/) the image built with Jib to have it available in your local Docker daemon.
 - Alternatively, instead of using a Docker daemon, you can run a local container registry, such as [Docker registry](https://docs.docker.com/registry/deploying/) or other repository managers, and point Jib to push to the local registry.
 
+### What would a Dockerfile for a Jib-built image look like?
+
+A Dockerfile that performs a Jib-like build is shown below:
+
+```Dockerfile
+FROM baseImage
+
+COPY libs /
+COPY snapshot-libs /
+COPY resources /
+COPY classes /
+COPY root /
```

Comment:
What's root?

Reply:
Oh, I could rename this to be more clear. We originally copied the extra directory into a folder called "root" in `jibExportDockerContext`.